### ISSUE TITLE
VR Support

### ISSUE THREAD

**Issue by mobettameta** [Santi871/DCSWaypointEditor#43]
I believe DCS Waypoint Editor doesn't work with VR right now unless I'm mistaken.  It sounded like the problem with it was that the location of the coordinates on screen is varying because the head position could be moving around.

I think this could be resolved a few ways:
1) Scan the full display on button press and look for the menu bar top left corner pixel by color search, then extract a fixed area based on that pixel location.

2) Have a different bind key for capture by VR.  This key would use the center of the view which would be pointed at the coords and capture a rectangular area around that.

I think maybe the resolution of the text might be a problem for the ocr, but i'm not sure on that.  Maybe we'd have to use VR zoom to zoom in real quick on the number.

This would be awesome to have as a lot of users are using VR.

**Comment by Penecruz** [comment]
Are any functions working in VR? What about the load to aircraft function? Log indicates that waypoints are being loaded however nothing is being entered into the WP’s in the aircraft. Tried on M2000 and F18.

**Comment by mobettameta** [comment]
> Are any functions working in VR? What about the load to aircraft function? Log indicates that waypoints are being loaded however nothing is being entered into the WP’s in the aircraft. Tried on M2000 and F18.

Yes, aircraft loading does work even in VR.  Not sure about M2000 and F18, but it definitely works on AV-8B.  Loading doesn't really have anything to do with VR.  The problem with VR is capturing data to the tool.

**Comment by SPAS79** [comment]
I have just tested v1.3.0 within VR and the Capture F10/F11 view seems to work, it reliably captures the center of the map screen. 

**The issue** with that is there is no "aiming mark" for the center of the screen. So you are always unsure if you are centered "enough" for good coordinates. A way around that might be using DCS Scratchpad which, in L/L mode, projects a white dot in the center of the screen which coincides with the point DCS Waypoint Editor seems to be looking at in F10/F11 capture mode. 

If such a center dot could be implemented within DCS wpEditor, that would be awesome!
